app/views/landing_page/blocks/_featured.html.erb

```diff
@@ -1,19 +1,27 @@
 <%
   add_view_stylesheet("landing_page/featured")
+  featured_child_content_classes = "featured_child"
+  block.image ? featured_child_content_classes << " featured__child--content-with-image" : featured_child_content_classes << " featured__child--content"
 %>
+
 <div class="featured">
-  <div class="featured__child featured__child--content">
+  <%= tag.div class: featured_child_content_classes do %>
     <% block.featured_content.each do |subblock| %>
-      <% options = { inverse: true } %>
+      <%
+        options = {}
+        options[:position] = "top" if subblock.type == "heading"
+      %>
       <%= render_block(subblock, options:) %>
     <% end %>
-  </div>
-  <div class="featured__child featured__child--image">
-    <%= picture_tag do %>
-      <%= tag.source srcset: "#{image_path(block.image.sources.desktop)}, #{image_path(block.image.sources.desktop_2x)} 2x", media: "(min-width: 769px)" %>
-      <%= tag.source srcset: "#{image_path(block.image.sources.tablet)}, #{image_path(block.image.sources.tablet_2x)} 2x", media: "(min-width: 641px) and (max-width: 768px)" %>
-      <%= tag.source srcset: "#{image_path(block.image.sources.mobile)}, #{image_path(block.image.sources.mobile_2x)} 2x", media: "(max-width: 640px)" %>
-      <%= image_tag(block.image.sources.desktop, alt: block.image.alt, class: "featured__image") %>
-    <% end %>
-  </div>
+  <% end %>
+  <% if block.image %>
+    <div class="featured__child featured__child--image">
+      <%= picture_tag do %>
+        <%= tag.source srcset: "#{image_path(block.image.sources.desktop)}, #{image_path(block.image.sources.desktop_2x)} 2x", media: "(min-width: 769px)" %>
+        <%= tag.source srcset: "#{image_path(block.image.sources.tablet)}, #{image_path(block.image.sources.tablet_2x)} 2x", media: "(min-width: 641px) and (max-width: 768px)" %>
+        <%= tag.source srcset: "#{image_path(block.image.sources.mobile)}, #{image_path(block.image.sources.mobile_2x)} 2x", media: "(max-width: 640px)" %>
+        <%= image_tag(block.image.sources.desktop, alt: block.image.alt, class: "featured__image") %>
```

Comment:
Do these images need alt text, or are they decorative like the hero block? If they're decorative, we should set a default empty string for the `alt` attribute, such as `image_alt_text = image.alt || ""`.

Reply:
Have pinged you a quick message on Slack about this one

Reply:
Thanks @jon-kirwan, have updated 👍 I've had a look at what happens when no alt text is provided and it renders `alt` rather than `alt=""`. At first I was a bit concerned that we'd need to have `alt=""` but it looks like just having [empty attribute syntax implicitly sets the value to ""](https://html.spec.whatwg.org/multipage/syntax.html#attributes-2)